CRM/Upgrade/Incremental/General.php

```diff
@@ -182,8 +182,12 @@ public static function updateMessageTemplate(&$message, $version) {
       1 => 'https://docs.civicrm.org/user/en/latest/email/message-templates/#modifying-system-workflow-message-templates',
       2 => '<ul>' . implode('', $messagesHtml) . '</ul>',
     ]);
-
-    $messageObj->updateTemplates();
+    if (version_compare($version, 5.26, '<')) {
```

Comment:
Is this backwards? I've always had trouble with that function.

Reply:
Yeh I'm 99.9999% sure your right there Dave

Reply:
`if (certainty_compare($version,99.9999, '<')) {`